### PR TITLE
ZIOS-7285: New camera keyboard permissions handler

### PR DESCRIPTION
**Pull request by nicorsm**
### Issues

There's a problem in the current implementation of `CameraKeyboardViewController`. In case the user doesn't allow to access their camera or photo library, the relative parts of the view remain empty, with a black camera view on the left side, and a grey collection view on the right side. The "open Camera Roll" button is also visible, even if we don't grant access to the library*.
Furthermore, you're continually prompted to give Wire permissions only for the camera when opening the camera keyboard. This is boring if you decide to definitely don't give permissions to the app.

_* which is ok from a technical side, because iOS 11 users still have the possibility to open the camera roll system dialog, but it's not clear to the user from a security point of view - "Why you still can access to my photos, even if I haven't authorized you?"._

### Solutions

I've implemented a new overlay view in case you don't have rights to access the camera and/or photo library, which replaces the respective section of the camera keyboard view. The "Settings" button inside this view automatically opens the Settings screen of the app.

testinitialstatelayoutsizecompact_cameraaccessgranted_ 320 216 2x

### CODE REVIEW

File: Wire-iOS/Resources/Base.lproj/Localizable.strings [hunk 1]
```diff
@@ -230,6 +230,12 @@
 "camera_access.denied.instruction" = "";
 "camera_access.denied.open_settings" = "Enable it in Wire Settings";
 
+// Camera and photos access, keyboard
+"keyboard_photos_access.denied.keyboard.settings" = "Settings";
+"keyboard_photos_access.denied.keyboard.camera_and_photos" = "Wire needs access to your\ncamera and photos.";
```
Comment (johnxnguyen) [inline Wire-iOS/Resources/Base.lproj/Localizable.strings]: I think the newline char could confuse some translators, they might not know where to put it or they could possibly leave it out altogether. I would suggest either to put a comment here for them, or consider to leave the newline out of the string and adjust the `UILabel` so it splits over two lines. What do you think?

**Comment by mat1th** [comment]
This fixes this issue https://github.com/wireapp/wire-ios/issues/243 :tada:

File: Wire-iOS/Resources/Base.lproj/Localizable.strings (hunk 1 above)
Comment (nicorsm) [reply Wire-iOS/Resources/Base.lproj/Localizable.strings]: There are other strings with this issue, like `"Start a conversation or\ncreate a group."`. I think that the reason why we put these line breaks is that we need to split the string into two lines with an equal amount of text for each line. For example, without a line break, the text could be "Wire needs access to your camera and (\n) photos." on bigger screens. Furthermore, Crowdin automatically checks for missing symbols like \n or %@. I'll add a comment anyway.

Comment (johnxnguyen) [reply Wire-iOS/Resources/Base.lproj/Localizable.strings]: Cool, I wasn't sure but good to know!

**Comment by nicorsm** [comment]
Tests are working fine locally. I'll merge the PR.

**Pull request merged**